STAT/Macros/QAtrendingFitExample.C

```diff
@@ -84,25 +110,44 @@ void loadMVAreaders(){
   AliNDFunctionInterface::LoadMVAReader(2,"TMVA_RegressionOutput.root","BDTRF25_8","dir_resolutionMIP");
   AliNDFunctionInterface::LoadMVAReader(3,"TMVA_RegressionOutput.root","BDTRF12_16","dir_resolutionMIP");
 }
-///
+
+/// Load array of regression  -used later in the array regression evaluation (mean, median, rms)
+///-------------------------------
 void loadMVAreadersBootstrap() {
   AliNDFunctionInterface::LoadMVAReaderArray(0,"TMVA_RegressionOutput.root","BDTRF12_16",".*resolutionMIP");
   AliNDFunctionInterface::LoadMVAReaderArray(1,"TMVA_RegressionOutput.root","BDTRF25_8",".*resolutionMIP");
   AliNDFunctionInterface::LoadMVAReaderArray(2,"TMVA_RegressionOutput.root","KNN",".*resolutionMIP");
 }
-
-void drawExamplePlots(){
+/// QAtrendingFitExample
+/// ----------------------
+void QAtrendingFitExample(){
+  /// 0.) Remove regression file
+  gSystem->Exec("rm -f TMVA_RegressionOutput.root");
```

Comment:
No. Use standard C functions for doing that, or standard ROOT ones, like `gSystem->Unlink()`.

Reply:
to be updated in new commit